### ISSUE TITLE
PATCH contact err 500

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#429]
in the following case I got an error 500 instead of 422 (if I'm right):

1. Open the same contact on two browser's tabs (or different devices)
2. Change a property on first tab
3. Without reloading, change same property on second tab

the PATCH request fail:

```json
{
	"errors": [{
		"code": 500,
		"description": "The server has either erred or is incapable of performing the requested operation.",
		"component": "contact",
		"values": 500,
		"message": "None",
		"property": "HTTPInternalServerError",
		"type": "Internal Server Error"
	}]
}
```

**Comment by sapiens-sapide** [comment]
Fix with commit #2972d2a.

**Comment by iamdey** [comment]
looks good
